frontend/public/components/utils/kebab.tsx

```diff
@@ -477,14 +477,20 @@ export const ResourceKebab = connectToModel((props: ResourceKebabProps) => {
     actions.map((a) => a(kindObj, resource, null, customData)),
     'hidden',
   );
+  let msg = '';
+  if (_.get(resource.metadata, 'deletionTimestamp')) {
+    msg = terminatingTooltip || t('public~Resource is being deleted');
+  }
   return (
     <Kebab
       options={options}
       key={resource.metadata.uid}
       isDisabled={
-        isDisabled !== undefined ? isDisabled : _.get(resource.metadata, 'deletionTimestamp')
+        // user can set isDisabled to true to disable kebab, but the terminating
+        // message should only be shown if the resource is actually terminating
+        isDisabled !== undefined ? isDisabled : !!_.get(resource.metadata, 'deletionTimestamp')
```

Comment:
```suggestion
        isDisabled ?? _.has(resource.metadata, 'deletionTimestamp')
```
Do we really need to disable the whole kebab menu? Sometimes users might want to edit the resource to remove the finalizers manually.

Reply:
IIRC this was the behavior earlier as well. Aman requested that when the kebab gets disabled, we should add a tooltip for contextual info.

Reply:
I'll remove the disable functionality from the Kebab, does that make sense? Since we do not have any use case for it at the moment.

Reply:
I guess if it was from before we can keep it as it is.